File: examples-sdk/go/302-rag-multi-source-fanin/main.go
// 302-rag-multi-source-fanin — Go SDK equivalent
// Compile: vil compile --from go --input 302-rag-multi-source-fanin/main.go --release
package main

import vil "github.com/OceanOS-id/vil-go"

func main() {
	s := vil.NewServer("rag-multi-source-fanin", 3111)
	s.Compile()
}

File: examples-sdk/zig/302-rag-multi-source-fanin/main.zig
// 302-rag-multi-source-fanin — Zig SDK equivalent
// Compile: vil compile --from zig --input 302-rag-multi-source-fanin/main.zig --release

const vil = @import("vil");

pub fn main() void {
    var server = vil.Server.init("rag-multi-source-fanin", 3111);
    server.compile();
}
